pages/login/LoginContainer.vue

```diff
@@ -0,0 +1,29 @@
+<template>
```

Comment:
Components like this should not be in the pages folder. The pages folder is basically the structure of the navigation routes. These should be in the components folder